### PR TITLE
Bump version to v0.10.0

### PR DESCRIPTION
**Pull request by JacobLiu-S**
This is v0.10.0 release version of mmhuman3d with updates in the latest three months.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ https://user-images.githubusercontent.com/62529255/144362861-e794b404-c48f-4ebe-
   A suite of differentiale visualization tools for human parametric model rendering (including part segmentation, depth map and point clouds) and conventional 2D/3D keypoints are available.
 
 ## News
+- 2022-09-30: MMHuman3D [v0.10.0](https://github.com/open-mmlab/mmhuman3d/releases/tag/v0.10.0) is released. Major updates include:
+  - Add webcam demo and real-time renderer
+  - Update dataloader to speed up training
+  - Add [balanced MSE](https://github.com/jiawei-ren/BalancedMSE) loss for imbalanced HMR training
```
Comment (caizhongang) [inline README.md:48]: Please also add new algorithms under `Supported methods`.

**Review by caizhongang — COMMENTED** [review]
In addition, please update the version to 0.10.0 in `version.py` in mmhuman3d/

**Comment by JacobLiu-S** [comment]
> In addition, please update the version to 0.10.0 in `version.py` in mmhuman3d/

When I changed the version to 0.10.0, I met lint and build errors. Solved by using python 3.8 in linting (because the package importlib-metadata is not compatible with flake8) and specifying mmcv-full in building (because mmcv-full 1.6.2 is not compatible with code in [here](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/__init__.py)).

File: README.md (hunk 1 above)
Comment (JacobLiu-S) [reply README.md:48]: > Please also add new algorithms under `Supported methods`.

This is already added in supported methods in one previous commits.

**Review by caizhongang — APPROVED** [review]

**Comment by OpenMMLab-Team** [comment]
Hi @JacobLiu-S ! First of all, we want to express our gratitude for your significant PR in the MMHuman3D project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution！❤

**Pull request merged**